scripts/jsorc/jaseci-template

```diff
@@ -139,6 +139,7 @@ metadata:
   name: jaseci
```

Comment:
Let's save our version of this file to `jaseci-template-experimental` and leave the original as is.